pkg/skaffold/deploy/kubectl_test.go

```diff
@@ -639,3 +640,55 @@ spec:
 		})
 	}
 }
+
+func TestGCSManifests(t *testing.T) {
+	tests := []struct {
+		description string
+		cfg         *latest.KubectlDeploy
+		commands    util.Command
+		shouldErr   bool
+		skipRender  bool
+	}{
+		{
+			description: "manifest from GCS",
+			cfg: &latest.KubectlDeploy{
+				Manifests: []string{"gs://dev/deployment.yaml"},
+			},
+			commands: testutil.
+				CmdRunOut(fmt.Sprintf("gsutil cp -r %s %s", "gs://dev/deployment.yaml", manifestTmpDir), "log").
+				AndRunOut("kubectl version --client -ojson", kubectlVersion112).
+				AndRunOut("kubectl --context kubecontext --namespace testNamespace create --dry-run -oyaml -f "+manifestTmpDir+"/deployment.yaml", deploymentWebYAML).
```

Comment:
please replace / with filepath.Join. I think your windows unit test is failing due to it.